Updating How Bad Standing Votes Work (#140)
ComputerScienceHouse/Constitution: articles.tex

Qualifying members may self-select into Active Membership by paying dues to the Financial Director and notifying the Evaluations Director.
\\*\\*

> Any Alumni Member in bad standing (\ref{Alumni Membership Selection}) may become an Active Member by notifying the Evaluations Director of their intent to participate, who will bring them up for vote at the next Evaluations meeting.

Reviewer: This line should be dropped as well.

> If a majority is in favor of the return, the Alumni is removed from bad standing.

Reviewer: ```suggestion
If a majority is in favor of the return, the Alumni is considered to be in good standing.
```
